README.rst

```diff
@@ -40,7 +40,9 @@ Scripts
 
 Scripts to simply common tasks:
 
-- compare.py_: compare a spec against two different versions of CrateDB.
+- compare.py_: compare measures read from two files
```

Comment:
shall we rename this to ``compare_measures`` or ``compare_results``? Whilst reading this I had to come back to this description a couple of times to re-review what the simple ``compare`` does